### ISSUE TITLE
Set clickable area for skinny buttons and tabs

### ISSUE THREAD

**Issue by maddyjoe** [RaspberryPiFoundation/editor-ui#288]
For the buttons and tabs (which should have been set up in  previous stories) but that follow [this updated component file](https://www.figma.com/file/2PjcTUsL1rHl1IaAht8yTm/Python-Editor---Dev-Handover?node-id=105%3A1910&t=V6TdGeLz73GcTZ1m-0)
we need to set the area for the interactive area/clickable area to be larger than the border (aka include the padding around the button) for accessibility reasons, to give a large enough minimum hit/touch area.

These skinny components are used -> in the tab sections for input/output, and in the buttons that appear in file panel.
